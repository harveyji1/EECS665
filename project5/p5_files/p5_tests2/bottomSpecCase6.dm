p : bool = true;
f : (x : int) bool { return true; }

main : () void {
    e : bool;
	e = (true + false) == 7;    // true + false is an error; the == is OK

}