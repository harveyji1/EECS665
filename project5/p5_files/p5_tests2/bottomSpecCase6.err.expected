FATAL [6,7]-[6,11]: Arithmetic operator applied to invalid operand
FATAL [6,14]-[6,19]: Arithmetic operator applied to invalid operand
Type Analysis Failed
